filebeat/processor/add_kubernetes_metadata/matchers.go

```diff
@@ -77,9 +77,9 @@ const containerIdLen = 64
 const podUIDPos = 5
 
 func (f *LogPathMatcher) MetadataIndex(event common.MapStr) string {
-	if value, ok := event["source"]; ok {
+	if value, ok := event["log"].(common.MapStr)["file"].(common.MapStr)["path"]; ok {
```

Comment:
This line probably breaks with the redis input as it does not have the fields in here.

Reply:
@alakahakai I think using `event.Get("log.file.path")` should be safe w.r.t. https://github.com/elastic/beats/issues/11543#issue-427061629.